atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueServiceTest.java

```diff
@@ -1192,8 +1249,10 @@ public void testGetRangeThrowsOnError() {
     @Test
     public void testGetRangeOfTimestampsThrowsOnError() {
         try {
-            keyValueService.getRangeOfTimestamps(TEST_NONEXISTING_TABLE, RangeRequest.all(), AtlasDbConstants.MAX_TS).hasNext();
-            Assert.fail("getRangeOfTimestamps must throw on failure");
+            assertFalse("getRangeOfTimestamps must throw on failure",
```

Comment:
This change misses the case where `hasNext` returns false no longer failing the test.  I think the right thing is to not test the return value and to remove the `catch` and replace with an `expected=RuntimeException.class` on the `Test` annotation.